### PR TITLE
fix(schedule): several bugfix

### PR DESCRIPTION
**Pull request by guowl3**
#### What type of PR is this?
type-bug

#### What this PR does / why we need it:
1. Create approval flow instance in  the community edition
2. Dispatch create schedule request to schedule service
3. Added concurrency protection measures for operations on Schedule
4. Execute sql plan in taskframework only in ob-cloud
#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/FlowInstanceController.java [hunk 1]
```diff
@@ -246,4 +260,42 @@ public SuccessResponse<PartitionPlanConfig> getPartitionPlan(@PathVariable Long
         return Responses.ok(this.partitionPlanScheduleService.getPartitionPlanByFlowInstanceId(id));
     }
 
+    private List<FlowInstanceDetailResp> dispatchCreateSchedule(CreateFlowInstanceReq createReq) {
```
Comment (yhilmare) [inline server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/FlowInstanceController.java:263]: please don't add any logic at controller layer

File: server/odc-service/src/main/java/com/oceanbase/odc/service/connection/model/ConnectionConfig.java [hunk 2]
```diff
@@ -311,7 +311,7 @@ public class ConnectionConfig
     @JsonIgnore
     private OBInstanceRoleType instanceRoleType;
 
-    @JsonIgnore
+    // @JsonIgnore
```
Comment (yhilmare) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/connection/model/ConnectionConfig.java:314]: we'd better not remove this

Comment (yhilmare) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/connection/model/ConnectionConfig.java:314]: you can add getter method if you want to return some other field by json

File: server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/FlowInstanceController.java (hunk 1 above)
Comment (guowl3) [reply server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/FlowInstanceController.java]: move to the scheduleService

File: server/odc-service/src/main/java/com/oceanbase/odc/service/connection/model/ConnectionConfig.java (hunk 2 above)
Comment (guowl3) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/connection/model/ConnectionConfig.java]: 😊modified to @Getter

**Review by yizhouxw — APPROVED** [review]
lgtm

**Review by kiko-art — APPROVED** [review]
lgtm

**Review by yhilmare — APPROVED** [review]
lgtm

**Pull request merged**